README.md

```diff
@@ -56,6 +56,18 @@ The boring generator introduces following generators:
 - Install SimpleForm: `rails generate boring:simple_form:install --css_framework=<css_framework>`
 - Install Devise: `rails generate boring:devise:install`
 
+**If you have Boring generators installed on your system, you can use it this way:**
+
+    $ boring generate boring:simple_form:install --css_framework=<css_framework>
```

Comment:
@luathn Just curious. We can explore this later as well. Can we use CLI as `boring simple_form:install` instead of `boring g boring:...`? 

Just want to make it more readable. Thoughts?

Reply:
Agh. Now I see it why we went with `boring generate <generator_name>`. We can ignore my above comment for now.